playwright-test/global-teardown.js

```diff
@@ -0,0 +1,15 @@
+// global-teardown.js
+const { bsLocal } = require('./fixtures');
+let localStopped = false;
+const {promisify} = require('util');
+const sleep = promisify(setTimeout);
+module.exports = async (config) => {
+  // Stop the Local instance after your test run is completed, i.e after driver.quit
+  bsLocal.stop(() => {
+    localStopped = true;
+    console.log('Stopped BrowserStackLocal')
```

Comment:
Is the local setup and teardown happening per worker?

Reply:
No, start and end of all the tests. It is not per worker. 
You can find detailed explanation here: https://playwright.dev/docs/test-advanced/#global-setup-and-teardown